models/converter.js

```diff
@@ -15,7 +15,7 @@ module.exports = class PDFConverterModel {
 
     options = Object.assign({
       format: 'A4',
-      waitUntil: 'networkidle2'
+      waitUntil: 'load'
```

Comment:
`waitUntil` defaults to `load` https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagegotourl-options

Reply:
Yes, but a) it's easier for us (me) to always provide a value with a default than to sometimes provide a value and sometimes not, and b) this protects us against a change in that default (even though this is unlikely)

Reply:
TBH, I'd be happy to never set the option, until a use case for "not load" appears, but mostly I just want this merged before our show and tell this afternoon.